apps/calendar/js/views/modify_event.js

```diff
@@ -357,74 +256,44 @@ ModifyEvent.prototype = {
       return;
     }
 
-    var self = this;
-    var provider;
-
-    this.store.providerFor(this.event, fetchProvider);
-
-    function fetchProvider(err, result) {
-      provider = result;
-      provider.eventCapabilities(
-        self.event.data,
-        verifyCaps
-      );
-    }
-
-    function verifyCaps(err, caps) {
-      if (err) {
-        return console.error('Error fetching capabilities for', self.event);
-      }
-
-      // safe-guard but should not ever happen.
-      if (caps[capability]) {
-        persistEvent();
-      }
-    }
-
-    function persistEvent() {
-      var list = self.element.classList;
+    // mark view as 'in progress' so we can style
+    // it via css during that time period
+    this.element.classList.add(this.PROGRESS);
 
-      // mark view as 'in progress' so we can style
-      // it via css during that time period
-      list.add(self.PROGRESS);
+    var moveDate = this.event.startDate;
 
-      var moveDate = self.event.startDate;
+    core.bridge[method](this.event).then(() => {
```

Comment:
it's a single async action and we need to handle success/error, what would be the benefit?